nautobot/docs/user-guide/administration/configuration/optional-settings.md

```diff
@@ -701,6 +701,14 @@ Toggle requiring authentication to view `/metrics`. See the [Prometheus Metrics]
 
 ---
 
+## METRICS_DISABLED_APPS
```

Comment:
This will need to be ported to `settings.yaml` for 2.2. Can you model this after `ALLOWED_HOSTS` and add information about how to set the environment variable?